docs/components/board/_index.md

```diff
@@ -1135,7 +1135,7 @@ For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/r
 
 ## `Analog` API
 
-### Read
+### ReadAnalogReader
```

Comment:
This should probably stay `Read` as well; analog now follows the gpio pin and digital interrupt pin pattern; it is no longer a reader only, but we have grouped Wrtie Analog in with it in the new board interface.